src/Stryker.Core/Stryker.Core/MutantFilters/IgnoreBlockMutantFilter.cs

```diff
@@ -30,7 +30,7 @@ public IEnumerable<Mutant> FilterMutants(IEnumerable<Mutant> mutants, IReadOnlyF
             var mutantsToIgnore = blockMutants.Where(blockMutant => mutants.Any(
                 m => m != blockMutant
                 && !_inactiveStatuses.Contains(m.ResultStatus)
-                && blockMutant.Span.Value.Contains(m.Span.Value)
+                && blockMutant.Mutation.OriginalNode.Span.Contains(m.Mutation.OriginalNode.Span)
```

Comment:
As seen here, `Span` properties was more harmful than helpful - direct access of a nullable's `Value` is a bad practice.